install: missing file operand

**Description**

slash-test-ami failure

Observed [here](https://openshift-gce-devel.appspot.com/build/origin-ci-test/pr-logs/pull/kubernetes-incubator_cri-o/1126/test_pull_request_crio_ami_fedora/185/) and [here](https://openshift-gce-devel.appspot.com/build/origin-ci-test/pr-logs/pull/kubernetes-incubator_cri-o/1126/test_pull_request_crio_ami_rhel/190/).

**Steps to reproduce the issue:**
1.  Run slash-test-ami
2. Tests fail on RHEL and Fedora

**Describe the results you received:**
The `make install` task fails:
```
TASK [install cri-o] ***********************************************************
task path: /go/src/github.com/kubernetes-incubator/cri-o/contrib/test/integration/build/cri-o.yml:22
fatal: [localhost]: FAILED! => {
    "changed": false, 
    "cmd": "/bin/make install", 
    "failed": true, 
    "rc": 2
}

STDOUT:
---
install -Z -D -m 755 bin/crio /usr/local/bin/crio
install -Z -D -m 755 bin/crioctl /usr/local/bin/crioctl
install -Z -D -m 755 bin/conmon /usr/local/libexec/crio/conmon
install -Z -D -m 755 bin/pause /usr/local/libexec/crio/pause
install -Z -d -m 755 /usr/local/share/man/man1
install -Z -d -m 755 /usr/local/share/man/man5
install -Z -d -m 755 /usr/local/share/man/man8
install -Z -m 644  -t /usr/local/share/man/man1

---

STDERR:
---
install: missing file operand
Try 'install --help' for more information.
make: *** [Makefile:152: install.man] Error 1
```

**Describe the results you expected:**

slash-test-ami should always pass

**Additional information you deem important (e.g. issue happens only occasionally):**

The prior task shows errors that maybe should have caused failure but did not?

The make (all) task:
```
TASK [build cri-o] *************************************************************
task path: /go/src/github.com/kubernetes-incubator/cri-o/contrib/test/integration/build/cri-o.yml:18
changed: [localhost] => {
    "changed": true, 
    "chdir": "/go/src/github.com/kubernetes-incubator/cri-o", 
    "failed": false, 
    "params": null, 
    "target": null
}

STDOUT:
---
go build -ldflags '-s -w -X main.gitCommit="a1c87f2788145a63e5bdd3aeddcbd365cc1af2e3" -X main.buildInfo=1510149261' -tags "seccomp btrfs_noversion  exclude_graphdriver_btrfs  selinux containers_image_ostree_stub" -o bin/crio github.com/kubernetes-incubator/cri-o/cmd/crio
/bin/make -C conmon
make[1]: Entering directory `/go/src/github.com/kubernetes-incubator/cri-o/conmon'
cc -std=c99 -Os -Wall -Wextra -I/usr/include/glib-2.0 -I/usr/lib64/glib-2.0/include     -c -o conmon.o conmon.c
cc -std=c99 -Os -Wall -Wextra -I/usr/include/glib-2.0 -I/usr/lib64/glib-2.0/include     -c -o cmsg.o cmsg.c
cc -o ../bin/conmon conmon.o cmsg.o -std=c99 -Os -Wall -Wextra -I/usr/include/glib-2.0 -I/usr/lib64/glib-2.0/include   -lglib-2.0  
make[1]: Leaving directory `/go/src/github.com/kubernetes-incubator/cri-o/conmon'
/bin/make -C pause
make[1]: Entering directory `/go/src/github.com/kubernetes-incubator/cri-o/pause'
cc -std=c99 -Os -Wall -Wextra -static   -c -o pause.o pause.c
cc -o ../bin/pause pause.o -std=c99 -Os -Wall -Wextra -static 
strip ../bin/pause
make[1]: Leaving directory `/go/src/github.com/kubernetes-incubator/cri-o/pause'
go build -ldflags '-s -w -X main.gitCommit="a1c87f2788145a63e5bdd3aeddcbd365cc1af2e3" -X main.buildInfo=1510149261' -tags "seccomp btrfs_noversion  exclude_graphdriver_btrfs  selinux containers_image_ostree_stub" -o bin/crioctl github.com/kubernetes-incubator/cri-o/cmd/crioctl
./bin/crio --config="" config --default > crio.conf
(go-md2man -in docs/crio.conf.5.md -out docs/crio.conf.5.tmp && touch docs/crio.conf.5.tmp && mv docs/crio.conf.5.tmp docs/crio.conf.5) || (/go/bin/go-md2man -in docs/crio.conf.5.md -out docs/crio.conf.5.tmp && touch docs/crio.conf.5.tmp && mv docs/crio.conf.5.tmp docs/crio.conf.5)
(go-md2man -in docs/crio.8.md -out docs/crio.8.tmp && touch docs/crio.8.tmp && mv docs/crio.8.tmp docs/crio.8) || (/go/bin/go-md2man -in docs/crio.8.md -out docs/crio.8.tmp && touch docs/crio.8.tmp && mv docs/crio.8.tmp docs/crio.8)
---

STDERR:
---
cmsg.c: In function ‘sendfd’:
cmsg.c:45:9: warning: missing braces around initializer [-Wmissing-braces]
  struct iovec iov[1] = {0};
         ^
cmsg.c:45:9: warning: (near initialization for ‘iov[0]’) [-Wmissing-braces]
cmsg.c: In function ‘recvfd’:
cmsg.c:91:9: warning: missing braces around initializer [-Wmissing-braces]
  struct iovec iov[1] = {0};
         ^
cmsg.c:91:9: warning: (near initialization for ‘iov[0]’) [-Wmissing-braces]
/bin/sh: go-md2man: command not found
/bin/sh: go-md2man: command not found
```

--- a/Makefile
+++ b/Makefile
@@ -94,7 +94,7 @@ ifneq ($(GOPATH),)
 	rm -f "$(GOPATH)/.gopathok"
 endif
 	rm -rf _output
-	rm -f docs/*.1 docs/*.5 docs/*.8
+	rm -f docs/*.5 docs/*.8
 	rm -fr test/testdata/redis-image
 	find . -name \*~ -delete
 	find . -name \#\* -delete
@@ -126,9 +126,6 @@ test-binaries: test/bin2img/bin2img test/copyimg/copyimg test/checkseccomp/check
 MANPAGES_MD := $(wildcard docs/*.md)
 MANPAGES    := $(MANPAGES_MD:%.md=%)
 
-docs/%.1: docs/%.1.md .gopathok
-	(go-md2man -in $< -out $@.tmp && touch $@.tmp && mv $@.tmp $@) || ($(GOPATH)/bin/go-md2man -in $< -out $@.tmp && touch $@.tmp && mv $@.tmp $@)
-
 docs/%.5: docs/%.5.md .gopathok
 	(go-md2man -in $< -out $@.tmp && touch $@.tmp && mv $@.tmp $@) || ($(GOPATH)/bin/go-md2man -in $< -out $@.tmp && touch $@.tmp && mv $@.tmp $@)
 
@@ -146,10 +143,8 @@ install.bin:
 	install ${SELINUXOPT} -D -m 755 bin/pause $(LIBEXECDIR)/crio/pause
 
 install.man:
-	install ${SELINUXOPT} -d -m 755 $(MANDIR)/man1
 	install ${SELINUXOPT} -d -m 755 $(MANDIR)/man5
 	install ${SELINUXOPT} -d -m 755 $(MANDIR)/man8
-	install ${SELINUXOPT} -m 644 $(filter %.1,$(MANPAGES)) -t $(MANDIR)/man1
 	install ${SELINUXOPT} -m 644 $(filter %.5,$(MANPAGES)) -t $(MANDIR)/man5
 	install ${SELINUXOPT} -m 644 $(filter %.8,$(MANPAGES)) -t $(MANDIR)/man8
 

